unit/goto-instrument/cover/cover_only.cpp

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************\
+
+Module: Tests for coverage instrumentation config object and filters
+
+Author: Diffblue Ltd, 2019
```

Comment:
2019 is not an author. We have a `Date: ` field in some versions of this header, but I'm not sure how useful that even is considering this is presumably going to be changed many times and we likely won't update the date field every time?